examples/tts/conf/vits.yaml

```diff
@@ -0,0 +1,256 @@
+name: "VITS"
+
+labels: [' ', '!', "'", '(', ')', ',', '-', '.', ':', ';', '?', 'a', 'b', 'c', 'd', 'e', 'f', 'g',
+         'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
```

Comment:
There are some parameters you can clean up like `labels` and `self._parser` in the model since they are unused.